src/annotator/config/index.js

```diff
@@ -19,6 +19,7 @@ function configFrom(window_) {
 
     annotations: settings.annotations,
     branding: settings.hostPageSetting('branding'),
+    onLayoutChange: settings.hostPageSetting('onLayoutChange'),
```

Comment:
Sidebar is probably redundant. it's not sidebarBranding, etc. Let's keep it the way it is 👍